cmd/bootstrap/cmd/finalize.go

```diff
@@ -663,3 +689,42 @@ func validateEpochConfig() error {
 	}
 	return nil
 }
+
+// validateOrPopulateEpochTimingConfig validates the epoch timing config flags. In case the
+// `flagUseDefaultEpochTargetEndTime` value has been set, the function derives the values for
+// `flagEpochTimingRefCounter`, `flagEpochTimingDuration`, and `flagEpochTimingRefTimestamp`
+// from the configuration. Otherwise, it enforces that compatible values for the respective parameters have been
+// specified (and errors otherwise). Therefore, after `validateOrPopulateEpochTimingConfig` ran,
+// the targeted end time for the epoch can be computed via `rootEpochTargetEndTime()`.
+// You can either let the tool choose default values, or specify a value for each config.
+func validateOrPopulateEpochTimingConfig() error {
+	if flagUseDefaultEpochTargetEndTime {
+		// No other flags may be set
+		if !(flagEpochTimingRefTimestamp == 0 && flagEpochTimingDuration == 0 && flagEpochTimingRefCounter == 0) {
+			return fmt.Errorf("invalid epoch timing config: cannot specify ANY of --epoch-timing-ref-counter, --epoch-timing-ref-timestamp, or --epoch-timing-duration if using default timing config")
+		}
+		flagEpochTimingRefCounter = flagEpochCounter
+		flagEpochTimingDuration = flagNumViewsInEpoch // default to 1 view/s
```

Comment:
I find one second per view pretty aggressive. Currently, we are running with `1.25 s/view`. Lowering this value significantly restricts Cruise Control's ability to catch up because it has 250ms less leverage per view to go faster.  

I guess according to your documentation above https://github.com/onflow/flow-go/blob/e766833d6e9f9dc7e9801adb8227873e85ee17ae/cmd/bootstrap/cmd/finalize.go#L107-L114 we only use this mode for benchnet or localnet, which typically run faster?